### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Name, Role and Value] [MAS4.1.2] [Incorrect Role is defined for vertical scroll bar of Iframe .]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#701]
**User Experience:**
User who rely on AT will not get proper information about the elements, if  role is not defined properly which provides the purpose of control.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Developer tool
**Attachments:**
![mas4 1 2_scroll bar](https://user-images.githubusercontent.com/33054846/32541281-3a04dcec-c495-11e7-8ffc-751e9c9b95cc.png)

**Repro Steps:**
1.	Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2.	Navigate to “report generation” Iframe  and open developer tool.
3.	Verify the  Role for the vertical scroll bar of Iframe.

**Current Behavior:**
Role is defined as Presentation for the vertical scroll bar, So that when user navigates in Item/Scan mode screen reader is announcing it as Image.

**Expected Behavior:**
Role should be defined as “Scroll bar” for the vertical scroll bar. So that user will get to know proper information about scroll bars.

**Note**
Issue is repro for the Entry Point page-Scroll bar 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.1.2
